pypesto/visualize/profile_cis.py

```diff
@@ -85,3 +87,105 @@ def profile_cis(
         ax.plot(ub, parameters_ind, "k--", marker="+")
 
     return ax
+
+
+def profile_nested_cis(
+        result: Result,
+        confidence_levels: Sequence[float] = (0.95, 0.9),
+        profile_indices: Sequence[int] = None,
+        profile_list: int = 0,
+        colors: Sequence = None,
+        ax: matplotlib.axes.Axes = None,
+        rotation: Literal["v", "h"] = "v",
+):
+    """
+    Plot approximate nested confidence intervals based on profiles.
+
+    Parameters
+    ----------
+    result:
+        The result object with profiling results.
+    confidence_levels:
+        The confidence levels in (0,1), which are translated to an approximate
+        threshold assuming a chi2 distribution, using
```

Comment:
Would it make sense to add the degrees of freedom for that chi2-dist. as a parameter?